packages/foam-vscode/src/features/open-daily-note.ts

```diff
@@ -98,7 +98,7 @@ async function focusDailyNote(dailyNotePath: string, isNewNote: boolean) {
 }
 
 async function pathExists(path: string) {
-  return fs.promises
+  return await fs.promises
```

Comment:
Just curious about the addition of `await` here. Is it required?  https://eslint.org/docs/rules/no-return-await

Reply:
@anku255 thanks for noticing this, I fixed this in [`9a68fe9` (#232)](https://github.com/foambubble/foam/pull/232/commits/9a68fe9f52537fd86eb91ebb5e2dca3751870482)